src/logic/addressBook/utils/__tests__/addressBookUtils.test.ts

```diff
@@ -1,16 +1,8 @@
-import { List } from 'immutable'
```

Comment:
create a different PR with tests for the recent additions (actions, reducers, `chainId`, util functions)